docs/source/grfn_api_v1.0.0.yaml

```diff
@@ -0,0 +1,211 @@
+swagger: "2.0"
+info:
+  description: "This document describes the API for interacting with AutoMATES GrFN."
+  version: "1.0.0"
+  title: "AutoMATES GrFN"
+  termsOfService: ""
+  contact:
+    email: "claytonm@arizona.edu"
+  license:
+    name: "TODO Apache 2.0"
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "TODOhost"
+basePath: "/v1"
+tags:
```

Comment:
@dpdicken I like the tags that you have so far, but can we add The following tags?
1. `/extract_dynamics`: tag that takes in a GrFN and returns the GrFN of the model dynamics
2. `/load_GrFN`: Takes in a GrFN JSON and loads that GrFN in a session to be executed
3. `/execute_loaded`: similar to `/execute` but runs new input values through a pre-loaded GrFN

We can probably de-prioritize (2) and (3) if those seem to require too much additional work.

Reply:
I think 1) sounds good and is necessary. I would argue that 2) and 3) sound good also, but I agree with you and don’t think we should do those initially as it’s a lot more functionality to create then an initial single execution call that can turn into these down the road.